fontFeatures/feaLib/Substitution.py

```diff
@@ -82,24 +131,16 @@ def asFeaAST(self):
             glyphref(self.replacement[0]),
             False,
         )
-    elif lut == 8:
+    elif lut in [5, 6, 7]: # GSUB 5, 6, 7 Different types of contextual substitutions
+        raise NotImplementedError("Use the Chain verb for this")
```

Comment:
Not sure this is quite true as FEE substitutions support pre/post context.

Reply:
That's true, but if we're getting _to this function_ to handle it, it's not supported.